lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -153,6 +153,8 @@ static void RegisterRequestCallbacks(DAP &dap) {
 
   // Testing requests
   dap.RegisterRequest<TestGetTargetBreakpointsRequestHandler>();
+  // WriteMemory request
+  dap.RegisterRequest<WriteMemoryRequestHandler>();
```

Comment:
Similarly to the comment in the CMake file, this should get inserted into the list of regular request handlers above, which are sorted alphabetically.